bears/js/JSHintBear.py

```diff
@@ -271,11 +271,10 @@ def generate_config(filename, file,
             - "outer" - check for variables defined in outer scopes as well
             - False - same as inner
             - True  - allow variable shadowing
-        :param check_unused_variables:
-            This option generates warnings when variables are defined but never
-            used. This can be set to ""vars"" to only check for variables, not
-            function parameters, or ""strict"" to check all variables and
-            parameters.
+        :param allow_unused_variables:
+            Allows when variables are defined but never used. This can be set
```

Comment:
Maybe you forgot to mention about the warnings that is produced? Not sure if this change looks good. Thoughts?